rules/Php80/AttributeDecorator/SensioParamConverterAttributeDecorator.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Php80\AttributeDecorator;
+
+use PhpParser\Node\Attribute;
+use Rector\Php80\Contract\AttributeDecoratorInterface;
+
+final class SensioParamConverterAttributeDecorator implements AttributeDecoratorInterface
+{
+    public function getAttributeName(): string
+    {
+        return 'Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter';
+    }
+
+    public function decorate(Attribute $attribute): void
+    {
+        // make first named arg silent, @see https://github.com/rectorphp/rector/issues/7352
+        $firstArg = $attribute->args[0];
+        $firstArg->name = null;
+    }
+}
```

Comment:
@javaDeveloperKid Btw, this is the fix for `ParamConverter` issue.  I've added interface so the fix for any attribute now is matter of few lines :+1: 

Give https://github.com/rectorphp/rector/issues/7369 a try :)

Reply:
This is the latest rector/rector that you should have there: https://github.com/rectorphp/rector/commit/d94a78124e59fb7b6f61e9dee0176a304c695788

Reply:
> For JMS\AccessorOrder it still doesn't work.

I does not. That's why I wrote "Give https://github.com/rectorphp/rector/issues/7369 a try :)"

Reply:
The 2nd commit is in here, rector-src.
Check only the composer version.

Reply:
I've just tried installing the latest dev Rector and it's right there:

![image](https://user-images.githubusercontent.com/924196/186683923-894237df-b526-405d-a2cc-1394f5a5de71.png)

Reply:
Yea, been there :) 

I usually delete `/vendor`, `composer.lock` (it caused us tremendous problems in the past) and install everything with `composer update`. That works far better than any other debugging.